controller/api/public/stat_summary_test.go

```diff
@@ -107,9 +107,7 @@ metadata:
 														LatencyMsP95: 123,
 														LatencyMsP99: 123,
 													},
-													TimeWindow:     "1m",
-													MeshedPodCount: 1,
```

Comment:
can you put these back, and update the pod fixture data:
```diff
diff --git a/controller/api/public/stat_summary_test.go b/controller/api/public/stat_summary_test.go
index 5bf337d..e34cb31 100644
--- a/controller/api/public/stat_summary_test.go
+++ b/controller/api/public/stat_summary_test.go
@@ -56,6 +56,8 @@ metadata:
     app: emoji-svc
   annotations:
     conduit.io/proxy-version: testinjectversion
+status:
+  phase: Running
 `, `
 apiVersion: v1
 kind: Pod
@@ -64,6 +66,8 @@ metadata:
   namespace: emojivoto
   labels:
     app: emoji-svc
+status:
+  phase: Running
 `,
                                },
                                promRes: model.Vector{
@@ -107,7 +111,9 @@ metadata:
                                                                                                                LatencyMsP95: 123,
                                                                                                                LatencyMsP99: 123,
                                                                                                        },
-                                                                                                       TimeWindow: "1m",
+                                                                                                       TimeWindow:     "1m",
+                                                                                                       MeshedPodCount: 1,
+                                                                                                       TotalPodCount:  2,
                                                                                                },
                                                                                        },
                                                                                },
```

Reply:
@siggy!!!!! I spent 30 minutes trying to figure out how to set status, to no avail. Thanks for this!